cli/share_create.go

```diff
@@ -171,22 +198,27 @@ func create(c *cli.Context) error {
 		}
 
 		if len(options.ValueOptions) > 0 && value == "" {
-			log.Warnf("Step input with 'value_options', should contain default value!")
+			log.Warnf("Step input with 'value_options' should contain a default value!")
 			fail("Missing default value for Step input (%s).", key)
 		}
 	}
 	if strings.Contains(*stepModel.Summary, "\n") {
 		log.Warnf("Step summary should be one line!")
 	}
 	if utf8.RuneCountInString(*stepModel.Summary) > maxSummaryLength {
-		log.Warnf("Step summary should contains maximum (%d) characters, actual: (%d)!", maxSummaryLength, utf8.RuneCountInString(*stepModel.Summary))
+		log.Warnf("Step summary should contain maximum (%d) characters, actual: (%d)!", maxSummaryLength, utf8.RuneCountInString(*stepModel.Summary))
 	}
 	log.Donef("step is valid")
 
 	// Copy step.yml to steplib
 	fmt.Println()
 	log.Infof("Integrating the Step into the Steplib...")
 
+	isStepNew, err := isStepNew(route, stepID)
```

Comment:
Quick question, why do we call `isStepNew` here if we only use the value at line 256?

Reply:
Because by that time we've created the directory and we couldn't detect if the step was new (because the check will always return false)